frontend/Movie_Pigeon/app/src/main/java/org/example/team_pigeon/movie_pigeon/GlobalReceiver.java

```diff
@@ -60,6 +60,11 @@ public void onReceive(Context context, Intent intent) {
                 sBuilder.execute(signInDetails);
                 break;
 
+            case "killRegistration":
+                Log.i(TAG, "Kill Registration received");
+                ((Activity) context).finish();
+                break;
+
             case "UserUpdate":
```

Comment:
to be consistent, this should be 'userUpdate'. Or change others to start with a capital letter.